core/src/cats/postgresql.cc

```diff
@@ -205,9 +205,9 @@ bool BareosDbPostgresql::OpenDatabase(JobControlRecord* jcr)
 
   // If connection fails, try at 5 sec intervals for 30 seconds.
   for (int retry = 0; retry < 6; retry++) {
-    db_handle_ = PQsetdbLogin(db_address_,   /* default = localhost */
-                              port,          /* default port */
-                              NULL,          /* pg options */
+    db_handle_ = PQsetdbLogin(db_address_, /* default = localhost */
+                              port,        /* default port */
+                              "-c client_min_messages=warning", /* pg options */
```

Comment:
I'm not really happy with doing this right here.

We already have code that sets several options during connect (see lines 239-245). I'd prefer doing a `SET client_min_messages TO WARNING;` there.

Reply:
Sure, I just force-pushed a new commit that implements it the other way.